python/taichi/lang/ndrange.py

```diff
@@ -3,7 +3,7 @@ def __init__(self, *args):
         args = list(args)
         for i in range(len(args)):
             if isinstance(args[i], list):
-                args[i] = list(args[i])
+                args[i] = tuple(args[i])
```

Comment:
OFT fix `ti.ndrange([a, b], [c, d])`.
I guess this is a typo.

Reply:
So in your example, it would become `args[0] = (0, [a, b])` in the old code?

Reply:
> So in your example, it would become args[0] = (0, [a, b]) in the old code?

Sounds like the case. It seems to me `ti.ndrange([...])` never worked before.